site/content/en/docs/Concepts/workflows.md

```diff
@@ -140,6 +140,27 @@ agc workflow run my-workflow --args inputs.json
 If the inputs file references local files, these will be synced with S3 and those files in S3 will be used when the workflow
 instance is run.
 
+#### `workflow optionFileUrl`
+
+An additional optionFileUrl can be provided using the 'o' or '--optionFileUrl' flag. For example:
+
+```shell
+agc workflow run my-workflow --optionFileUrl optionFile.json
+```
+
+If the option file references a local file, it will be synced with S3 and the file in S3 will be used when the workflow
+instance is run. This is only for use with engines that run in server mode (e.g. Cromwell). It will be ignored if not using an engine  running in server mode.
+
+#### `workflow engineOptions`
+
+An additional property 'engineOptions' can be provided using the 'e' or '--engineOptions' flag. For example:
```

Comment:
Since `engineOptions` is part of `workflowEngineParams`, specifying both flags is currently undefined behavior. Presumably we want `--engineOptions` to override the value used by same named key of the file specified with  `--optionFileUrl`, but we should call out that behavior so customers know how they interact together

Reply:
This is what the understanding is at the moment, but I'm creating a new issue to look into this more since it looks like they are not actually equivalents for server mode and head node engines.